### ISSUE TITLE
Crash on iOS 9 with JPush 2.2.0 which installed from CocoaPods

### ISSUE THREAD

**Issue by nixzhu** [jpush/jpush-ios-sdk-pod#11]
```
dyld: Library not loaded: /System/Library/Frameworks/UserNotifications.framework/UserNotifications
  Referenced from: /var/containers/Bundle/Application/428E2D9E-A519-4502-8E57-43B3DDDA253D/Yep.app/Yep
  Reason: image not found
```
You guys need to modify https://github.com/jpush/jpush-ios-sdk-pod/blob/master/JPush.podspec.json
Put UserNotifications in weak_frameworks
https://guides.cocoapods.org/syntax/podspec.html#weak_frameworks

**Comment by huangminlinux** [comment]
yeah ,you are right,,  I will fix this version ， but now the cocoapods server error occurred

**Comment by huangminlinux** [comment]
you can adjust build setting -> other linker flags -> 
```
-framework
"UserNotifications"
```
to 

```
-weak_framework
"UserNotifications"
```
to fix it

**Comment by huangminlinux** [comment]
fixed in 2.2.0.1 version, thx for your Feedback
